jsapp/js/account/security/accessLogs/accessLogs.query.ts

```diff
@@ -50,3 +50,23 @@ export default function useAccessLogsQuery(
     refetchOnWindowFocus: true,
   });
 }
+
+/**
+ * Starts the exporting process of the access logs.
+ * @returns {Promise<void>} A promise that starts the export.
+ */
+export const startAccessLogsExport = (): Promise<void> =>
+  new Promise<void>((resolve, reject) => {
```

Comment:
Do I understand this correctly - `fetchPost` is returning a promise, but you need to wrap it in `new Promise` here to handle the error, right?

Reply:
it can be simplified without wrapping, this should be equivalent:
```js
fetchPost(endpoints.ACCESS_LOGS_EXPORT_URL, {})
  .catch((error) => {
        const failResponse: FailResponse = {
          status: 500,
          statusText:
            error.message || 'An error occurred while exporting the logs.',
        };
        throw failResponse;
      });